docs/man/git-lfs-migrate.1.ronn

```diff
@@ -271,7 +272,8 @@ $ git lfs migrate import --no-rewrite test.zip *.mp3 *.psd
 With a specified commit message:
 
 ```
-$ git lfs migrate import --no-rewrite -m "Import .zip, .mp3, .psd files" \
+$ git lfs migrate import --no-rewrite \
+  -m "Import test.zip, .mp3, .psd files in root of repo" \
   test.zip *.mpd *.psd
```

Comment:
These actual arguments are still different from what commit message says.

- commit message: `.mp3`
- actual: `*.mpd`

This issue had been described in the second section of https://github.com/git-lfs/git-lfs/pull/3424#discussion_r245543300